Classes/Command/RemoveInactiveOrdersCommand.php

```diff
@@ -67,17 +73,32 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 );
             } catch (\Throwable $exception) {
                 $output->writeln('Could not cancel the order ' . $inactiveOrder->getUid() . ' using cancellation service!');
-                // anyway make sure to remove the order!
+                // Anyway make sure to remove the order!
                 $cancellationService->getPersistenceManager()->remove($inactiveOrder);
             }
-            $progressBar->advance(1);
+
+            $progressBar->advance();
         }
+
         $progressBar->finish();
+
         $cancellationService->getPersistenceManager()->persistAll();
+
         $output->writeln('Clear caches for affected facilities list views...');
+
         foreach ($affectedFacilities as $facilityUid => $_) {
             CacheUtility::clearPageCachesForPagesWithCurrentFacility($facilityUid);
         }
         return 0;
     }
+
+    public function getCancellationService(): CancellationService
+    {
+        return GeneralUtility::makeInstance(CancellationService::class);
```

Comment:
CancellationService uses PersistenceManager. It's Extbase. Don't use Extbase Methods in Commands.